lib/network.go

```diff
@@ -2132,6 +2141,12 @@ func (msg *MsgDeSoHeader) EncodeHeaderVersion2(preSignature bool) ([]byte, error
 	// The Nonce and ExtraNonce fields are unused in version 2. We skip them
 	// during both encoding and decoding.
 
+	// TxnConnectStatusByIndexHash
+	if msg.TxnConnectStatusByIndexHash == nil {
```

Comment:
Does this need to be gated on some migration?

Reply:
No migration needed here. The existence of the filed is gated by the header version value = 2